### DISCUSSION TITLE
[Feature] Add GPS location to photos extracted from gpx file

### DISCUSSION THREAD

**Discussion by Tuporat** [immich-app/immich#11108] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Hi, it would be helpful to add an option where Immich automatically adds GPS coordinates to the images extracted from a GPX track based on the recording time. The matching factor would be the time of recording. 

I remember that Lightroom offers this option. The background to this wish is that I often take photos with an action cam that has no GPS. I have the track recorded on my cell phone at the same time.
Best regards and thank you!

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by radh21301** [comment]
In the meanwhile, if you want a free alternative to Lightroom that does this: Digikam - https://userbase.kde.org/Digikam/Geotagging

**Comment by bo0tzz** [comment]
#6831
